client/ping.go

```diff
@@ -18,31 +23,39 @@ import (
 // may be returned if the daemon is in an unhealthy state, but returns errors
 // for other non-success status codes, failing to connect to the API, or failing
 // to parse the API response.
-func (cli *Client) Ping(ctx context.Context) (types.Ping, error) {
+func (cli *Client) Ping(ctx context.Context, requestFeatures bool) (types.Ping, error) {
```

Comment:
Instead of a boolean, should we allow passing a specific endpoint version here?

```suggestion
func (cli *Client) Ping(ctx context.Context, featuresVersion string) (types.Ping, error) {
```